### PR TITLE
virtmedia fixes

### PR DESCRIPTION
**Pull request by derekhiggins**
Two patches from upstream image for virtmedia install

### CODE REVIEW

File: Dockerfile.ocp [hunk 1]
```diff
@@ -2,14 +2,18 @@ FROM ironic-builder AS builder
...
+## that gets installed by grub2-efi-x64 (/boot/efi/EFI/centos/grubx64.efi)
+## looks for grub.cnf in /EFI/centos, ironic puts it in /boot/grub
 RUN if [ $(uname -m) = "x86_64" ]; then \
-      yum install -y gcc git make genisoimage xz-devel grub2 grub2-efi-x64 shim dosfstools mtools && \
+      yum install -y gcc git make genisoimage xz-devel grub2 grub2-efi-x64-modules shim dosfstools mtools && \
       dd bs=1024 count=3200 if=/dev/zero of=esp.img && \
       mkfs.msdos -F 12 -n 'ESP_IMAGE' ./esp.img && \
       mmd -i esp.img EFI && \
       mmd -i esp.img EFI/BOOT && \
+      grub2-mkimage -C xz -O x86_64-efi -p /boot/grub -o /tmp/grubx64.efi boot linux search normal configfile part_gpt btrfs ext2 fat iso9660 loopback test keystatus gfxmenu regexp probe efi_gop efi_uga all_video gfxterm font scsi echo read ls cat png jpeg halt reboot && \
```
Comment (derekhiggins) [inline Dockerfile.ocp:14]: Note: this line in the upstream patch included the linuxefi module, this appears to not be needed in RHEL8 (or available)

Comment (elfosardo) [reply Dockerfile.ocp:14]: the linuxefi is not needed in general, it's long gone from upstream grub and it was kept just for compatibility with old versions, please also see https://github.com/metal3-io/ironic-image/pull/128

**Pull request merged**